### PR TITLE
Fix apostrophe-S

### PR DESCRIPTION
**Pull request by matthias-ccri**
Small fix here...

Apostrophe-S is not plural. I dare say it's _never_ plural.

Correct:
I have many dogs. :+1: :dog2: 

Incorrect:
I have many dog's. :thinking:

### CODE REVIEW

**Review by grahamsmith — APPROVED** [review]

**Comment by grahamsmith** [comment]
Spot on! Please rebase and we can get this merged. 👍

**Comment by grahamsmith** [comment]
Did a quick check - the file has been moved to here but the fixes are still valid https://docs.rocket.chat/guides/user-guides/message-actions

**Comment by matthias-ccri** [comment]
Rebased! Caught a few more, while I'm in here.
Also an ["its"](https://www.dictionary.com/e/its-vs-its/) issue.

Thanks!

File: guides/user-guides/message-actions.md [hunk 1]
```diff
@@ -30,17 +30,17 @@ Starred messages are per chat room and are not global; each room has its list of
 
 ## Pin messages
 
-Pin a message to a channel to make that message easily accessible to all user's on the channel. To pin a message, select the **Pin Message** option in the message actions menu. Remove already pinned messages by selecting **Remove Pin**.
+Pin a message to a channel to make that message easily accessible to all users on the channel. To pin a message, select the **Pin Message** option in the message actions menu. Remove already pinned messages by selecting **Remove Pin**.
 
-_The administrator of a Rocket.Chat team sets the permissions for who can pin messages in a channel. By default, only administrators, moderators, and owners of the channel can pin messages ._
+_The administrator of a Rocket.Chat team sets the permissions for who can pin messages in a channel. By default, only administrators, moderators, and owners of the channel can pin messages._
```
Comment (grahamsmith) [inline guides/user-guides/message-actions.md:35]: If we are being picky then the comma before the 'and' is unnecessary in my opinion. I won't block the approval of this PR on this though.

**Review by grahamsmith — APPROVED** [review]
Approved! Just check what I spotted and if agreed then might be good to fix too.

File: guides/user-guides/message-actions.md (hunk 1 above)
Comment (matthias-ccri) [reply guides/user-guides/message-actions.md:35]: That's the Oxford Comma Style. Personally, I'm a fan of that style, but in any case... I didn't touch the comma, so it's unrelated to this PR.

**Pull request merged**